prescient.el

```diff
@@ -193,6 +193,15 @@ partially matched candidates, but candidates in each group will
 still be sorted like normal."
   :type 'boolean)
 
+(defcustom prescient-use-char-folding t
+  "Whether certain literal filtering methods use character folding.
+
+This affects the `literal' and `literal-prefix' filtering methods.
+
+See also the customizable variables `char-fold-include',
+`char-fold-exclude', and `char-fold-symmetric'."
```

Comment:
Those seem to be not available in Emacs 27, maybe mention that they are available starting with Emacs 28?

Reply:
> Those seem to be not available in Emacs 27, maybe mention that they are available starting with Emacs 28?

The [news file `NEWS.27`](https://git.savannah.gnu.org/cgit/emacs.git/tree/etc/NEWS.27#n1647) and [this post by Mickey Petersen](https://masteringemacs.net/article/whats-new-in-emacs-27-1) list them as appearing in 27.1, and I have them in a version installed from Guix.

Do you think that

> In Emacs versions 27 and greater, see also the customizable variables `char-fold-include`, `char-fold-exclude`, and `char-fold-symmetric`.

is a good way to phrase it?

Reply:
That is weird I don't have them in my Emacs 27.1. The phrase you suggested looks good to me otherwise.